provision/testdata/list_flavors.stdout.txt

```diff
@@ -1,5 +1,6 @@
 INFO: Available configuration flavors:
 INFO: openshift-4.5-openstack:	Red Hat OpenShift Container Platform 4.5 on OpenStack
+INFO: openshift-4.6-openstack:	Red Hat OpenShift Container Platform 4.6 on OpenStack
```

Comment:
Once you change the order numbers, this order will also change, and you will just need to swap the above two.